src/components/DropZone/utils/index.ts

```diff
@@ -54,3 +54,9 @@ function accepts(file: File, acceptedFiles: string | string[] | undefined) {
 function isDragEvent(event: DropZoneEvent): event is DragEvent {
   return dragEvents.indexOf(event.type) > 0;
 }
+
+function isChangeEvent(
+  event: DropZoneEvent,
+): event is React.ChangeEvent<HTMLInputElement> {
+  return 'target' in event;
```

Comment:
Should we use `hasOwnProperty` since `in` will return true if `target` is anywhere in the prototype chain?